spec/features/workflow/clear_backdate_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+RSpec.feature "Clear backdate" do
+  scenario do
+    given_there_is_an_editable_backdated_edition
+    when_i_visit_the_summary_page
+    i_can_see_the_date_the_content_has_been_backdated_to
+    when_i_click_to_edit_the_backdate
+    and_i_click_clear_backdate
+    then_i_see_the_content_is_no_longer_backdated
+  end
+
+  def given_there_is_an_editable_backdated_edition
+    @backdated_to = Time.current.yesterday
+    @edition = create(:edition, backdated_to: @backdated_to)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def i_can_see_the_date_the_content_has_been_backdated_to
+    expect(page).to have_content(@backdated_to.strftime("%-d %B %Y"))
```

Comment:
Is there any we can more specific for this and the other check?

I wonder if we can be checking for `expect(page).to have_content("#{backdate_title} #{backdate_time}")` as I think that would match?

Reply:
Good idea.